src/FactSystem/FactControls/FactStepper.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.5
+import QtQuick.Controls 1.2
+import QtQuick.Controls.Styles 1.2
+import QtQuick.Dialogs         1.2
+
+import QGroundControl.ScreenTools   1.0
+import QGroundControl.FactSystem 1.0
+import QGroundControl.Palette 1.0
+import QGroundControl.Controls 1.0
+
+Item {
+    property bool enabled: true
+    property alias fact: factInput.fact
+
+    property string valueText:          fact ? fact.valueString : "N/A"
```

Comment:
Qml coding style has internal properties use same C++ style ```_``` prefix to identify public versus private variables. So if these are internal use only they should be ```_valueText``` for example/